### ISSUE TITLE
Double quote - still a bug

### ISSUE THREAD

**Issue by adaur** [wbb/WysiBB#93]
Hi,

It has been reported twice, but never fixed and closed by the author... any reason?

https://github.com/wbb/WysiBB/issues/69
https://github.com/wbb/WysiBB/issues/24

Cheers

**Comment by Youpain** [comment]
Hello,

for my, i use phpbb3, when i quote a author, i need a double quote.

I change in jquery.wysibb.min.js

				quote : {
					title: CURLANG.quote,
					buttonHTML: '<span class="fonticon ve-tlb-quote1">\uE00c</span>',
					hotkey: 'ctrl+shift+3',
					//subInsert: true,
					transform : { 
						'<blockquote>{SELTEXT}</blockquote>':"[quote]{SELTEXT}[/quote]",
						'<blockquote><cite>{AUTHOR} a écrit:</cite><br>{SELTEXT}</blockquote>':"[quote={AUTHOR}]{SELTEXT}[/quote]"
					}


Attention, use link to direction your file :+1: 


<script src="wysibb/jquery.wysibb.min.js"></script>
<link rel="stylesheet" href="wysibb/theme/default/wbbtheme.css" />

**Comment by Karlingue** [comment]
Please fix !

**Comment by wbb** [comment]
http://www.gitfund.org/wbb/wysibb

**Comment by Karlingue** [comment]
rofl ...

**Comment by adaur** [comment]
Still, it is a bug... what is this link supposed to mean? That I should pay you to fix this?

**Comment by wbb** [comment]
> Still, it is a bug... what is this link supposed to mean? That I should pay you to fix this?

The link is show priorities for issues. Priorities that have top(100%) rank will be fixed asap. Everyone can change priorities by donations.

**Comment by Karlingue** [comment]
This is silly.

**Comment by javakind** [comment]
> This is silly.
I don't think so. By that prioritity shema we can see bugs that are realy have top priotities. 

Also anybody can fix bug and enjoy.

**Comment by demon101** [comment]
My project also affected by this bug. I need nested spoiler tags.

**Comment by demon101** [comment]
вообще как-то со вложенностью тегов все плохо
например вот такой код [u] =) test test [/u] со смайлом становится просто текстом, если открыть исходный код, а потом опять вернуться к визуальном виду. Притом через раз.
Те, в визуальном виде, делаешь текст например жирным, потом в этот жирный текст вставляешь смайл, он нормально отображается, переходишь в коды и обратно, нет смайла

**Comment by adaur** [comment]
Just upping this (:

**Comment by Machou** [comment]
+1

**Comment by error0608** [comment]
+1

**Comment by Kanobiya** [comment]
**Youpain** - After three or four quotes, double quotes - are meaningless.
Error - [/ quote] [/ quote]
